doc/source/reference/c-api/array.rst

```diff
@@ -584,25 +584,26 @@ From other objects
     did not have the _ARRAY_ macro namespace in them. That form
     of the constant names is deprecated in 1.7.
 
-.. c:macro:: NPY_ARRAY_NOTSWAPPED
+    .. c:macro:: NPY_ARRAY_NOTSWAPPED
```

Comment:
These are un-indented on purpose so that the macros can be linked to from elsewhere